shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -506,30 +514,32 @@ \subsubsection*{The Rewards that are received by each Delegator per Epoch}
 \noindent
 The rewards that are received by an individual delegator are directly proportional to the
 stake that the delegator has placed in the StakePool, calculated from the net rewards to the
-StakePool.
-
-\clearpage
-\subsubsection*{The Rewards that are received by each StakePool Owner per Epoch}
-
-\begin{figure}[h!]
-\begin{center}
-\begin{tabular}{||l|p{6cm}|l||}
-  \hline \hline
-\textbf{Parameter}  & \textbf{Definition} & \textbf{Calculated as} \\\hline
-$R^\textit{Owner}$ & The Reward to the Owner(s) & $R^{\textit{Net}} \times {\textit{Pool}}^{\textit{Margin}} $ \\\hline
-$\textit{nOwners}$ & The total number of Owner(s) of the StakePool, $\textit{nOwners} \ge 1$  & \\\hline
-$O^{Rewards}$ & The \ada{} rewards that are received by owner $O$ & $R^\textit{Owner} \div \textit{nOwners}$ \\\hline
-\hline
-\end{tabular}
-\end{center}
-\caption{The Rewards that are received by a specific Owner $O$.}
-\end{figure}
-
-\noindent
-In the Incentivised TestNet, the owner rewards are divided equally among the
-owners, regardless of the proportion of stake that they pledge.  The assumption
-is that, in practice, each owner will pledge an equal amount of stake, but this
-is not enforced by the implementation.
+StakePool.  This applies to the owner stake as well as to independent delegators.
+If there are multiple owners, rewards will subsequently be distributed between them according to an external
+agreement that will not be governed by or enforced by the TestNet.
```

Comment:
should we state explicitly that the agreement is only between the owners in that case?

Reply:
The agreement could be between any parties.